rustyms-py/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rustyms_py"
```

Comment:
I have always used 'kebab case' for package names (meaning I would use `rustyms-py` here.

Reply:
Fixed!